src/pages/ReimbursementAccount/ReimbursementAccountPage.js

```diff
@@ -245,12 +245,7 @@ class ReimbursementAccountPage extends React.Component {
                         />
                     )}
                     {currentStep === CONST.BANK_ACCOUNT.STEP.VALIDATION && (
-                        <ValidationStep
-                            achData={this.props.reimbursementAccount.achData}
```

Comment:
Why don't we need these props anymore? Just double checking before i merge :)

Reply:
ah seems those props were removed in the `main` branch. I removed the extra spaces I inserted unintentionally in that file to avoid conflicts.